extensions/amp-story/0.1/amp-story.css

```diff
@@ -123,6 +123,20 @@ amp-story[standalone]:fullscreen {
   background-color: rgba(0, 0, 0, 0.2) !important;
 }
 
+.i-amphtml-story-system-reset,
+.i-amphtml-story-system-reset * {
+  border: none !important;
```

Comment:
Did you try `all`?  Do we know if that's a reasonable approach or not?

Reply:
As discussed offline, ``all`` wouldn't let us override properties.